lib/components/tree.component.ts

```diff
@@ -10,7 +10,7 @@ import { KEYS } from '../constants/keys';
 import * as _ from 'lodash';
 
 @Component({
-  selector: 'Tree',
+  selector: 'Tree, tree',
```

Comment:
Hey,
I prefer to allow using kebab-case names, but issue a "deprecated" warning (using lib/deprecated) if someone is using the Capitalized selectors.
If you have time to update the PR that would be nice.